modules/benchmark/R/load.netcdf.R

```diff
@@ -0,0 +1,29 @@
+load.netcdf <- function(path,vars,units,newunits){
+  
+  print(paste("Years: ",start.year," - ",end.year),sep="")
+  result <- list()
+  for(ncfile in ncfiles) {
+    nc <- nc_open(ncfile)
+    for(i in i:length(vars)){
+      v <- vars[i]
+      u1 <- units[i]
+      u2 <- newunits[i]
+      if(v %in% c(names(nc$var),names(nc$dim))){
+        newresult <- ncvar_get(nc, v)
+        newresult <- ud.convert(newresult, u1, u2)
+        result[[v]] <- abind(result[[v]], newresult)
+      } else if (!(v %in% names(nc$var))){
+        logger.warn(paste(v, "missing in", ncfile))
+      }
+    }
+    nc_close(nc)
+  }
+  
+  print(paste("----- Mean ", variables, " : ",
+              lapply(result, mean, na.rm = TRUE)))
+  print(paste("----- Median ", variables, ": ",
+              lapply(result, median, na.rm = TRUE)))
+  
```

Comment:
In the [`read.output` function in the biocro-module2 branch](https://github.com/PecanProject/pecan/compare/biocro-module2#diff-dd16c68e01b2961df28b59d0bcd4e023R143) (not yet committed to master) I wrote a variant on this to a) use the `logger.info` function and b) provide rounded numbers:

```r
logger.info(variables, 
  print(paste("----- Mean ", variables, " : ",		+              "Mean:", lapply(result, function(x) signif(mean(x, na.rm = TRUE), 3)),
              lapply(result, mean, na.rm = TRUE)))		+              "Median:", lapply(result, function(x) signif(mean(x, na.rm = TRUE), 3)))
```